pkg/vere/vere.h

```diff
@@ -1238,6 +1244,12 @@
         u3_auto*
         u3_lick_io_init(u3_pier* pir_u);
 
+    /**  mdns, routing for the local network
+    **/
+      /* mdns_init(): initialize mdns.
+      */
+        void
+        mdns_init(uint16_t port, char* our, mdns_cb* cb, void* context);
```

Comment:
Where should I put this signature? `vere.h` seems wrong but don't really know where it should go.